pkg/mosn/mosn.go

```diff
@@ -87,8 +87,6 @@ func (m *Mosn) upgradeCheck() {
 		log.StartLogger.Infof("[mosn] [NewMosn] active reconfiguring")
 		// set Mosn Active_Reconfiguring
 		store.SetMosnState(store.Active_Reconfiguring)
-		// parse MOSNConfig again
-		c = configmanager.Load(configmanager.GetConfigPath())
```

Comment:
keep this，use new c to m.Config

Reply:
Done, plz review again.

Reply:
We should reset `m.config` directly whether or not `InheritOldMosnconfig` is true.

Reply:
Done.